x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/convert/ToDatetime.java

```diff
@@ -40,10 +41,39 @@ public class ToDatetime extends AbstractConvertFunction {
         Map.entry(INTEGER, ToLongFromIntEvaluator.Factory::new) // CastIntToLongEvaluator would be a candidate, but not MV'd
     );
 
-    @FunctionInfo(returnType = "date", description = "Converts an input value to a date value.")
+    @FunctionInfo(
+        returnType = "date",
+        description = "Converts an input value to a date value.\n"
+            + "A string will only be successfully converted if it's respecting the format `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'`. "
+            + "To convert dates in other formats, use <<esql-date_parse>>.",
+        examples = {
+            @Example(
+                file = "date",
+                tag = "to_datetime-str",
+                note = "Note that in this example, the last value in the source multi-valued field has not been converted. "
```

Comment:
Can you use `"""` to get the same? It'll have new lines in it, but I think that's ok? Or we could remove it when we copy it.